fenix/app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -71,4 +71,9 @@ object FeatureFlags {
      * Enables the notification pre permission prompt.
      */
     const val notificationPrePermissionPromptEnabled = true
+
+    /**
+     * Enables the redesigned onboarding.
+     */
+    const val junoOnboardingEnabled = false
```

Comment:
❓ Should this be Config.channel.isNightlyOrDebug?

Reply:
Yeah it can be, my thinking was to keep it disabled till the last patch is in.

Reply:
I think both are fine.  I would probably enable this in Nightly/Debug once there are for onboarding dialogs so we can make sure it's disabled during UI testing.

Reply:
yeah, will update it to `nightlyOrDebug` when the last patch is in 👍 
Good point about UI testing, will keep that in mind as this goes into beta/release.

Reply:
That's good to know, thanks! 
And for the usage, could we then just call `FxNimbus.features.junoOnboarding.value().enabled` instead of
using  `lazyfeatureFlagPreference()`?

Reply:
Yes, exactly. We don't need to check two values then - just the one.

Reply:
But we still want to experiment on it though.  Don't I need the feature flag to be true and then use the Nimbus Flag to experiment?